<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolTip">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/spacebg.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:0.175325 rgba(170, 85, 127, 255), stop:0.321429 rgba(122, 122, 255, 255), stop:0.353896 rgba(112, 112, 255, 255), stop:0.542289 rgba(218, 167, 127, 255), stop:0.79 rgba(170, 0, 255, 255), stop:0.967532 rgba(255, 158, 158, 255), stop:1 rgba(0, 0, 127, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>411</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0, 0);
font: 48pt &quot;MS UI Gothic&quot;;
</string>
    </property>
    <property name="text">
     <string>StarExpress</string>
    </property>
   </widget>
   <widget class="QPushButton" name="login_pushbutton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>500</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:0.175325 rgba(170, 85, 127, 255), stop:0.321429 rgba(122, 122, 255, 255), stop:0.353896 rgba(112, 112, 255, 255), stop:0.542289 rgba(218, 167, 127, 255), stop:0.79 rgba(170, 0, 255, 255), stop:0.967532 rgba(255, 158, 158, 255), stop:1 rgba(0, 0, 127, 255));</string>
    </property>
    <property name="text">
     <string>войти</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signup_pushbutton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>570</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">selection-color: rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string>зарегистрироваться</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="img/bg.qrc"/>
 </resources>
 <connections/>
</ui>
